markdown/developers/guidelines/index.md

```diff
@@ -0,0 +1,80 @@
+- [Use the template](requirements/use_the_template.md): All nf-core pipelines must be built using the nf-core template.
+- [Software licence](requirements/mit_licence.md): Pipelines must open source, released with the MIT licence.
+- [Bundled documentation](requirements/docs.md): Pipeline documentation must be hosted on the nf-core website.
+- [Docker support](requirements/docker.md): Software must be bundled using Docker and versioned.
+- [Continuous integration testing](requirements/ci_testing.md): Pipelines must run CI tests.
+- [Semantic versioning](requirements/semantic_versioning.md): Pipelines must use stable release tags.
+- [Standardised parameters](requirements/parameters.md): Strive to have standardised usage.
+- [Single command](requirements/single_command.md): Pipelines should run in a single command.
+- [Keywords](requirements/keywords.md): Excellent documentation and GitHub repository keywords.
+- [Pass lint tests](requirements/linting.md): The pipeline must not have any failures in the `nf-core lint` tests.
+- [Credits and Acknowledgements](requirements/acknowledgements.md): Pipelines must properly acknowledge prior work.
+- [Minimum inputs](requirements/minimum_inputs.md): Pipelines should be able to run with as little input as possible.
+- [Use nf-core git branches](requirements/git_branches.md): Use `master`, `dev` and `TEMPLATE`.
+
+## Recommendations
+
+All nf-core pipelines _should_ follow the following guidelines, if possible / appropriate:
+
+- [Use Bioconda](recommendations/bioconda.md): Package software using bioconda and biocontainers.
+- [File formats](recommendations/file_formats.md): Use community accepted modern file formats such as `CRAM`.
+- [DOIs](recommendations/dois.md): Pipelines should have digital object identifiers (DOIs).
+- [Cloud compatible](recommendations/cloud_compatible.md): Pipelines should be tested on cloud computing environments.
+
+## If the guidelines don't fit
+
+We appreciate that the above guidelines are relatively rigid and may not be for everyone.
+If that's the case, there is still a lot of ways that you can get involved with nf-core!
+
+We hope that the nf-core best practices, tooling and community are helpful for anyone building Nextflow pipelines, even if they are not a good fit for being listed as official nf-core pipelines.
+You are very welcome to use the helper tools and collaborate on modules / subworkflows / ideas.
+Indeed, numerous pipelines outside of nf-core now extensively use and contribute back to [nf-core/modules](https://github.com/nf-core/modules).
+
+If using nf-core tools and especially the template, please don't call your pipeline `nf-core/<yourpipeline>`.
+Please say that your pipeline _"uses"_ nf-core rather than rather than _"is"_ nf-core.
+Remember that you can generate a pipeline with `nf-core create` that excludes nf-core branding.
+Citation and acknowledgement of the work that goes into these tools and templates is welcome.
```

Comment:
```suggestion
- If using nf-core tools and especially the template, please don't call your pipeline `nf-core/<yourpipeline>`.
- Please say that your pipeline _"uses"_ nf-core rather than rather than _"is"_ nf-core.
- Remember that you can generate a pipeline with `nf-core create` that excludes nf-core branding.
- Citation and acknowledgement of the work that goes into these tools and templates is welcome.
```

Better as bullets to act as a 'checklist'?

Reply:
I don't get why I can't see this comment on the review "Files changed" tab and I can't actually see what you've changed in the suggested comment...

Ignoring as I want to merge for bytesize, but please open a new PR / complain at me until I do something 👍🏻